src/en/charms-deploying.md

```diff
@@ -1,67 +1,88 @@
 the use of charms (the magic bits of code that make things just work). These
-charms can be fetched from the charm store, stored in a local repository, or if
-you are feeling clever, written by you. Just as there are different series of
-Ubuntu ('precise', 'raring', etc), so there needs to be different series of
-charms to take into account any subtle changes in the underlying OS. For the
-most part you can forget about this, as Juju will always try to apply the most
-relevant charm, so deploying can be straightforward and easy.
+charms can exist in the [Charm Store](https://jujucharms.com/store) or on the
+file system (previously downloaded from the store or written locally).
+
+Charms use the concept of *series* analogous as to how Juju does with Ubuntu
+series ('trusty', 'xenial', etc). For the most part, this is transparent as
+Juju will use the most relevant charm to ensure things "just work". This makes
+deploying services with Juju fun and easy.
 
 
-# Deploying from the Charm Store
+## Deploying from the charm store
 
-In most cases, you will want to deploy charms by fetching them directly from the
-Charm Store. This ensures that you get the relevant, up to date version of the
-charm and "everything just works". To deploy a charm like this you can just
-specify:
+Typically, services are deployed using the online charms. This ensures that
+you get the latest version of the charm. To deploy in this way:
 
 ```bash
 juju deploy mysql
 ```
 
-Running this will do exactly what you expect - fetch the latest Juju charm for
-the series you are running and then use the bootstrap environment to initiate a
-new instance and deploy MySQL
+This will create a machine and use the latest online MySQL charm (for your
+default series) to deploy a MySQL service.
+
+!!! Note: The default series can be configured at a model level (see
+[Configuring model](./models-config.html)). In the absence of this setting,
+the default is to use the Ubuntu version running on the Juju client (i.e. where
+the Juju commands are being invoked).
 
-Juju usefully supports a system of namespaces that means you can actually deploy
-charms from a variety of sources. The default source is the charm store. The
-above command is the same as running:
+Assuming that the Xenial series charm exists and was used above, an equivalent
+command is:
 
 ```bash
-juju deploy cs:precise/mysql
+juju deploy cs:xenial/mysql
 ```
 
-which follows the format:
+Where 'cs' denotes the charm store.
 
-```nohighlight
-<repository>:<series>/<service>
-```
+### Channel support	
 
+The charm store offers charms in different stages of development. Such stages
+are called *channels*.
 
-# Deploying from a local repository
+```bash
+juju deploy mysql --channel channel_name
+```
 
-There are many cases when you may wish to deploy charms from a local filesytem
-source rather than the charm store:
+Such a channel will be used if the charm's revision is:
 
```

Comment:
this doesn't make a lot of sense to me. It isn't your fault - I looked through the doc we were given and that doesn't make sense either. It doesn't help that we don't mention revisions anywhere else